.changeset/brave-pandas-add.md

```diff
@@ -0,0 +1,10 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+fix: provide a proper error message when using `getCloudflareContext` in static routes
+
+`getCloudflareContext` can't be used in static routes, currently a misleading error
```

Comment:
note: at some poin we should add an "How To" for the dev flow at `https://opennext.js.org/cloudflare/howtos/...`

Reply:
Sure, I can work on it whenever you want, just let me know how to prioritize that 👍

Reply:
Maybe create an issue (there is documentation column on the board) and we can prioritize during the weekly?

Reply:
Issue created: https://github.com/opennextjs/docs/issues/67